specification/_types/aggregations/bucket.ts

```diff
@@ -731,6 +731,25 @@ export class ReverseNestedAggregation extends BucketAggregationBase {
   path?: Field
 }
 
+export class RandomSamplerAggregation extends BucketAggregationBase {
+  /**
+   * The probability that a document will be included in the aggregated data.
+   * Must be greater than 0, less than 0.5, or exactly 1.
+   * The lower the probability, the fewer documents are matched.
+   */
+  probability: float
```

Comment:
Docs say float, but server says [double](https://github.com/elastic/elasticsearch/blob/701ed61fd31aed60e9e1ae7ba5ab4d24f266f928/server/src/main/java/org/elasticsearch/search/aggregations/bucket/sampler/random/RandomSamplerAggregationBuilder.java#L58)

Reply:
Thanks, I should have checked that part, sorry. Done in [919eb559 (#2944)](https://github.com/elastic/elasticsearch-specification/pull/2944/commits/919eb559939ea8cf1e421e80a4e0ff6991422b8b).